pkg/crc/preflight/preflight.go

```diff
@@ -151,10 +153,13 @@ func doCleanUpPreflightChecks(checks []Check) error {
 		}
 		err := check.doCleanUp()
 		if err != nil {
-			return err
+			// If an error occurs in a cleanup function
+			// we log/collect it and  move to the  next
+			logging.Debug(err)
+			mErr.Collect(err)
 		}
 	}
-	return nil
+	return mErr
```

Comment:
I think we want to return nil if there was no error, and `mErr` if there was at least one error?

Reply:
Won't an empty `mErr` be seen sa an error anyway? On retrospect, I'm not sure it was a good idea when I pushed for `multierror` to implement the `error` interface, a `ToError()` method was probably better :)
Apart from this, /lgtm

Reply:
Yes, empty `mErr` is seen as an error. Should we then add a `ToError()` method in `multierror`?

Reply:
I would go for a quick work-around in this PR, and a follow-up PR if you want to clean that up with better API

Reply:
For now checking at the end if `mErr` is empty and then return nil or error based on that..
```diff
-       return nil
+       if len(mErr.Errors) <= 0 {
+               return nil
+       }
+       return goerrors.New(mErr.Error())
```